### DISCUSSION TITLE
with Subtitles it is showing No additional information... ?

### DISCUSSION THREAD

**Discussion by shubhamserverfront** [laravel/nova-issues#6277] [Q&A] [answered]
- Laravel Version: 10.10
- Nova Version: 4.0
- PHP Version: 8.2.0
- Database Driver & Version:
- Operating System and Version: ubuntu
- Browser type and version: chrome
- Reproduction Repository: https://github.com/###/###

### Description:
BelongsTo::make('Account')
        ->sortable()
        ->searchable()
        ->rules('required')
        ->required()
        ->showOnPreview()
       ->withSubtitles(function ($value) {
                return $value->email; 
           }),

### Detailed steps to reproduce the issue on a fresh Nova installation:
#this is my transaction resource

i have this code but when i try to get email in Subtitles it is showing "No additional information..."

And if i follow officals docs it say use

public function subtitle() { return "Author: {$this->user->name}"; } If i use this then subtitle is not coming

How to use subtitle and how to get email in subtitle?

**Comment by webard** [comment, marked as answer]
`withSubtitles()` method does not accept any arguments. Your `Account` resource should define `subtitle()` method. I used it yesterday, it works perfectly.

**Reply by shubhamserverfront** [reply]
public function account(): BelongsTo
    {
        return $this->belongsTo(Account::class);
    }
#this is my model

![subtitle](https://github.com/laravel/nova-issues/assets/155707482/4011790d-5c36-4e8e-820d-a5e728aa6cb1)


            BelongsTo::make('Account')
                ->sortable()
                ->searchable()
                ->rules('required')
                ->required()
                ->showOnPreview()
                }),
 #This is my resource file?
In this i want to add subtitle how to do it ?
below account number i want a subtitle should come

**Reply by webard** [reply]
Here add '-withSubtitles()' and in Account resource define 'subtitle()' and return data which you want to display.

**Reply by shubhamserverfront** [reply]
Thanks man it work
